gluten-ut/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -172,7 +171,15 @@ trait GlutenTestsTrait extends SparkFunSuite with ExpressionEvalHelper with Glut
       _spark.createDataFrame(_spark.sparkContext.parallelize(empData), schema)
     }
     val resultDF = df.select(Column(expression))
-    val result = resultDF.collect()
+    val result = if (justEvalExpr) {
+      try {
+        expression.eval(inputRow)
+      } catch {
+        case e: Exception => fail(s"Exception evaluating $expression", e)
+      }
+    } else {
+      resultDF.collect()
```

Comment:
Does `resultDF.collect()` raise error? Can we fix that?

Reply:
Thanks for your comment! I think it is just a testing issue, not related to the main code functionality. Let's respect vanilla spark's testing approach.